<?xml version="1.0"?>
<launch>
      <arg name="drone_name" default="anafi"/>
      <param name="use_sim_time" value="false"/>
      <group ns="$(arg drone_name)">
        <node name="sphinx_stability_frame_publisher_node" pkg="sphinx_with_gazebo" type="sphinx_publish_copter_stability_axes.py" respawn="true" output="screen">
        <param name="drone_name" value="$(arg drone_name)"/>
        </node>
      </group>  
</launch>